neurom/apps/morph_stats.py

```diff
@@ -136,6 +138,16 @@ def generate_flattened_dict(headers, results):
 }
 
 
+def full_config():
+    '''Returns a config with all features, all modes, all neurite types'''
```

Comment:
It's not really 'full' if it doesn't include some of the features, no?

Reply:
Are there some missing features ?

Reply:
ah, I see now